src/app/containers/ATIAnalytics/amp/index.jsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 import { string } from 'prop-types';
+import xssJsonStringify from '#lib/utilities/xssJsonStringify';
 import getAmpAnalyticsJson from './ampAnalyticsJson';
 
 const JsonInlinedScript = data => (
   <script
     type="application/json"
     /* eslint-disable-next-line react/no-danger */
-    dangerouslySetInnerHTML={{ __html: JSON.stringify(data) }}
+    dangerouslySetInnerHTML={{ __html: xssJsonStringify(data) }}
```

Comment:
same as https://github.com/bbc/simorgh/pull/4051/files#r331890213 and https://github.com/bbc/simorgh/pull/4051/files#r331888321 because it we are outputting JSON and not html